src/Castle.Core.Tests/ProxyGenerationOptionsTestCase.cs

```diff
@@ -23,7 +24,9 @@ namespace Castle.DynamicProxy.Tests
 
 	using NUnit.Framework;
 
-	[TestFixture]
+	using DescriptionAttribute = System.ComponentModel.DescriptionAttribute;
+
+    [TestFixture]

```

Comment:
Yep, will do. Did not notice it was tabs. I'll figure out how to set my VS without changing my global settings (all of my other projects are spaces :P)